modules/claims_api/lib/bgs_service/claimant_service.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  class ClaimantService < ClaimsApi::LocalBGS
```

Comment:
Should this be `ClaimantWebService` (and `claimant_web_service.rb`)?

Reply:
ClaimantService aligns more with the other files here. Typically we name it after the first part of the Bean, less the "Bean". Or at least we _have_. May be a bit misleading because the VDC ones and it looks like we better align with the latter part of the bean.

[BenefitClaimService](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/claims_api/lib/bgs_service/benefit_claim_service.rb)
[BenefitClaimWebService](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/claims_api/lib/bgs_service/benefit_claim_web_service.rb)

If it seems more meaningful to do it after the latter so it's a 1:1 alignment, glad to discuss that as an option, we'd just have to formulate a plan for reworking a bit of them. 6 one, half dozen the other.

Reply:
> ClaimantService aligns more with the other files here.

For the purposes of this PR, I think we should stay with `ClaimantService` then. And I agree - if we want to change the naming pattern, we may want to do it in a separate task after fully considering the tradeoffs.